ncsa-common-incore/src/main/java/edu/illinois/ncsa/incore/dao/IncoreDatasetDao.java

```diff
@@ -376,4 +467,17 @@ public List<Dataset> findByCreatorEmailAndTitleLikeAndDeleted(String email, Stri
         return null;
     }
 
+    // Helper method for testing with a local instance
+    public String getToken() {
+        String bearerToken = null;
+        try {
+            String userHome = System.getProperty("user.home");
+            byte[] encoded = Files.readAllBytes(Paths.get(userHome + "/.incore/.a0309ff368531845cbeef9a617fcf11361da600ec8cc1467b2a579cb52890f85_token"));
```

Comment:
will this file always exist? Would it make sense to see if there is a file in ~/.incore that ends with _token?

Reply:
You are right, it may not exist if the user didn't log into the server, let me fix that. I can also make this more generalized if I use the same method pyincore uses to create the token name. pyincore takes the sha 256 of the server URL and use that to create the token name in the ".incore" folder. Since the user needs to set the server URL as a parameter in datawolf.properties if I they are using the IN-CORE file storage, I can look for the file with the 256 sha of that name.